modules/meeting/app/controllers/meetings_controller.rb

```diff
@@ -110,6 +111,26 @@ def update
 
   private
 
+  def load_query
+    query = ParamsToQueryService.new(
+      Meeting,
+      current_user
+    ).call(params)
+
+    if @project
+      query.where("project_id", '=', @project.id)
+    else
+      # global meetings page should only list future meetings
+      query.where("time", "=", "future")
```

Comment:
Since `future` is a magic string, extracting it into a constant seems advisable.

Reply:
yep, will do